plrust/build

```diff
@@ -3,17 +3,21 @@ set -xe
 
 if [ -z "$STD_TARGETS" ]; then
     # if none specified, always build for these two targets
-    STD_TARGETS="x86_64-postgres-linux-gnu aarch64-postgres-linux-gnu"
+    export STD_TARGETS="x86_64-postgres-linux-gnu aarch64-postgres-linux-gnu"
```

Comment:
This should instead be explicitly passed to `./run install`, to indicate the dependency.

Reply:
This was how it historically was, but it was undone as part of #165 changes.

Reply:
It needs to be passed to clean too, now that cross compilation exists.